custom_model_runner/datarobot_drum/drum/language_predictors/r_predictor/r_predictor.py

```diff
@@ -104,3 +104,68 @@ def predict(self, input_filename):
                 logger.error(error_message)
                 raise DrumCommonException(error_message)
         return py_data_object
+
+    def predict_unstructured(self, data, **kwargs):
+        def _r_is_character(r_val):
```

Comment:
we'll need to make sure those internal functions are covered with tests currently its hard to (they are nested), only end-to-end calling predict_unstructured.

Reply:
added TODO